tests/autoyast/clone.pm

```diff
@@ -19,11 +19,27 @@
 use strict;
 use base 'console_yasttest';
 use testapi;
+use version_utils 'is_sle';
 
 sub run {
     my $self = shift;
     assert_script_run 'rm -f /root/autoinst.xml';
-    assert_script_run 'yast2 --ncurses clone_system', 400;
+    script_run('(yast2 --ncurses clone_system;echo ZYPPER-DONE) | tee /dev/$serialdev', 0);
+    # precompile regexes
+    my $zypper_done  = qr/^ZYPPER-DONE/m;
+    my $clone_checks = [$zypper_done];
+    my $timeout      = 400;
+    if (is_sle('15+') && check_screen 'autoyast2-install-accept', 30) {
+        assert_screen 'autoyast2-install-accept';
+        send_key 'alt-i';
+    }
+    my $out = wait_serial($clone_checks, $timeout);
+    while ($out) {
+        if ($out =~ $zypper_done) {
+            last;
+        }
+        $out = wait_serial($clone_checks, $timeout);
+    }
```

Comment:
Seems no need to use regexes here, it is simple one line to check if yast2 clone system is finished:
`wait_serial('yast2-clone_system-status-0', 400) || die "'yast2 clone system' didn't finish";`

Reply:
Updated code and verified log: http://openqa-apac1.suse.de/tests/3011#step/clone/4